templates_ark/1_Segment_Image_Data.ipynb

```diff
@@ -41,6 +41,24 @@
     "If you're interested in directly interfacing with Google Drive, consult the documentation [here](https://ark-analysis.readthedocs.io/en/latest/_rtd/google_docs_usage.html)."
```

Comment:
<p>The tiff_dir is <code>input_data</code> here but <code>raw</code> in the data download function. How come the notebook didn't fail?  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/angelolab/ark-analysis/pull/685/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
`raw` is where the actual output of the Hugging Face dataset gets put into. So `tiff_dir` is `data/example_dataset/raw/{hugging-face-path}/input_data`. 

`input_data` is the name of the zip file that contains all of the fovs. `hugging-face-path` is a rather long series of folders. 

Currently, on my computer it is: `downloads/extracted/c32fe76b55fb47f6130713504f1103a1aaf9b04ac4c888612add911a54c5de54`. But this can change over time.

Reply:
We don't need that hugging face alphanumeric subfolder though, right? Can we just save the data directly in the directory structure that's in the design doc?

Reply:
No not directly, however we could set the cache to the default location which is `/.cache/huggingface/datasets`. And then copy the files over to `data/example_dataset/input_files`.

Reply:
If each user has a different path to the actual data, it's going to make it hard for subsequent notebooks to use data that was previously downloaded. We should do whatever gymnastics we need to do so that when the user looks at the example data folder, it's structured normally how they would actually name it.